packages/components/src/server/body.client.tsx

```diff
@@ -6,15 +6,21 @@ import { cn } from '../cn';
 
 export interface BodyProps
   extends DetailedHTMLProps<HtmlHTMLAttributes<HTMLBodyElement>, HTMLBodyElement> {
-  lightOnlyPages?: string[];
+  lightOnlyPages: string[];
 }
 
 export const Body: FC<BodyProps> = ({ lightOnlyPages, children, className, ...rest }) => {
   const pathname = usePathname();
-  const isLightOnlyPage = lightOnlyPages?.includes(pathname);
+  const isLightOnlyPage = lightOnlyPages.includes(pathname);
 
   return (
-    <body className={cn(className, isLightOnlyPage && 'light text-green-1000')} {...rest}>
+    <body
+      className={cn(
+        className,
+        isLightOnlyPage && 'light text-green-1000 [&_#h-navmenu-container]:max-w-[1392px]',
```

Comment:
The width and light mode forcing shouldn't be connected IMHO.

We already have dark mode pages with narrower width, take a look at the comment in #1995 please.

Reply:
https://github.com/the-guild-org/docs/pull/1995/files#diff-16d12c62838ebc54876aac48e6300e7e706a736387f9eb2ad78247b40904d4a3R19-R29

Reply:
So I'd add `<HiveLayoutConfig widths="landing-narrow" />` to that Modules page.

Reply:
It was, but I fixed it in the Components v9 / Nextra 4 branch. Hive main is "outdated" compared to Components.

Case Studies pages is narrow, but dark-friendly.

I realize it makes the code more complex, but I think we're getting asked about dark more by users every now and then, so decoupling this would help us have more dark mode pages.